tests/modules/test_SeleniumExecutor.py

```diff
@@ -1149,6 +1165,9 @@ def test_build_script(self):
         with open(__dir__() + "/../selenium/generated_from_requests.py") as sample:
             sample_contents = sample.readlines()
 
+        # with open(__dir__() + "/../selenium/generated_from_requests.py", "w+") as sample:
```

Comment:
Here it server real purpose to be able to uncomment in the future to ease debugging of script generating process.